.github/workflows/push-dev-image.yml

```diff
@@ -5,6 +5,10 @@ name: Push development image
 on:
   workflow_dispatch:
     inputs:
+      pr:
+        description: 'PR number (if deploying a PR)'
```

Comment:
maybe to be clearer we could have something like this:

> PR number to build (if empty use above branch)

but I still don't like that much either...